File: scratch/clear_db.py
import os
from supabase import create_client
from dotenv import load_dotenv

load_dotenv('.env.local')

SUPABASE_URL = os.environ.get('NEXT_PUBLIC_SUPABASE_URL')
SUPABASE_KEY = os.environ.get('SUPABASE_SERVICE_ROLE_KEY')

sb = create_client(SUPABASE_URL, SUPABASE_KEY)
# Clear all guests by deleting everything
try:
    res = sb.table('guests').delete().neq('id', '00000000-0000-0000-0000-000000000000').execute()
    # Also clear system logs to have a fresh start
    sb.table('system_logs').delete().neq('id', '00000000-0000-0000-0000-000000000000').execute()
    print("Success: Database cleared.")
except Exception as e:
    print(f"Error clearing database: {e}")

File: scratch/clear_db.mjs
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config({ path: '.env.local' });

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
const supabaseKey = process.env.SUPABASE_SERVICE_ROLE_KEY || process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;
const supabase = createClient(supabaseUrl, supabaseKey);

async function clear() {
  console.log('Clearing ALL tables completely...');
  
  // Clean jobs
  const { error: e1 } = await supabase.from('jobs').delete().neq('status', 'placeholder_status_that_does_not_exist');
  
  // Clean guests (phone is a string)
  const { error: e2 } = await supabase.from('guests').delete().neq('phone', 'placeholder_phone');
  
  // Clean logs (level is a string)
  const { error: e3 } = await supabase.from('system_logs').delete().neq('level', 'placeholder_level');
  
  if (e1 || e2 || e3) {
    console.error('Error clearing:', e1 || e2 || e3);
  } else {
    console.log('Success: Database completely formatted.');
  }
}
clear();
